### ISSUE TITLE
Question: Deckstats.xml after syncing with Hearthstats

### ISSUE THREAD

**Issue by culinko** [Epix37/Hearthstone-Deck-Tracker#1026]
Hey, I synced all of my decks and matches into the Hearthstats yesterday and found out something weird. Only one of my priest decks got assigned a <HearthStatsDeckId> tag, while other decks got created with a new ID. Can I fix this manually by removing them and replacing the <HearthStatsDeckId> tag manually? Does it cause any harm because of the different <DeckId>?

For example this is a deck initially created by Deck Tracker:

    <Deck>
      <DeckId>bce5813f-a624-48c8-b06a-868b9728cf86</DeckId>
      <Games>
      ....
      </Games>
      <HearthStatsDeckId />
      <Name>Oil Rogue</Name>
    </Deck>

And this deck got created after I synced with Hearthstats.

    <Deck>
      <DeckId>ce5bc079-34fe-479b-b995-bc2c25648570</DeckId>
      <Games />
      <HearthStatsDeckId>539214</HearthStatsDeckId>
      <Name>Oil Rogue</Name>
    </Deck>

**Comment by Epix37** [comment]
I don't quite understand the problem, could you rephrase that?

**Comment by culinko** [comment]
Instead of changing <HearthStatsDeckId /> to <HearthStatsDeckId>539214</HearthStatsDeckId>, the tracker added new Decks with new DeckIDs at the end of the DeckStats.xml. I am asking if I manually remove the empty decks at the end of the file and replace the <HearthStatsDeckId /> with <HearthStatsDeckId>539214</HearthStatsDeckId>, if it will break something.

**Comment by Epix37** [comment]
No idea honestly. Backup and try :).

**Comment by culinko** [comment]
Haven't tried anything yet, just found out that bce5813f-a624-48c8-b06a-868b9728cf86 is the "Main ID (current v1.3)" and ce5bc079-34fe-479b-b995-bc2c25648570 that got created recently is the ID of v1.0. The priest deck is the only deck that has just one version, so that's why it didn't create anything new for that deck. In DeckStats.xml, the matches are grouped under the current (newest) Id, so that's why it created a new Deck there. Is it correct, that the HearthStatsDeckId is bound to v1.0 of the deck?
